src/include/catalog/catalog_entry/rel_group_catalog_entry.h

```diff
@@ -19,7 +19,7 @@ class RelGroupCatalogEntry final : public TableCatalogEntry {
     //===--------------------------------------------------------------------===//
     bool isParent(common::table_id_t tableID) override;
     common::TableType getTableType() const override { return common::TableType::REL_GROUP; }
-    std::vector<common::table_id_t>& getRelTableIDs() { return relTableIDs; }
+    const std::vector<common::table_id_t>& getRelTableIDsRef() const { return relTableIDs; }
```

Comment:
let's still call this getRelTableIDs(). No need to call it getRef if you are returning const&